### PR TITLE
Update ECFR-XML-User-Guide.md

### PR DESCRIPTION
**Pull request by JnyJny**
Fixed broken markdown section headers by inserting a space after octothorps.

### CODE REVIEW

**Comment by michaelblyons** [comment]
You have a bunch of other whitespace changes that came along for the ride. Was that intentional?

**Comment by JnyJny** [comment]
Nope, those were definitely not intentional! The idea was to help not make more work :(

**Comment by datbossg31030** [comment]
New email Datbossg702@gmail.com

**Review by jonquandt — COMMENTED** [review]
I am going to reject this one and just make the manual changes for the octothorp/heading fixes separately. Based on the diffs, I don't expect a ton of issues in accepting this, but I'd rather constrain the changes to just the few lines where needed.

File: ECFR-XML-User-Guide.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ The purpose of this document is to provide an overview of e-CFR XML files and as
 
 **Q. What is the data set available for the e-CFR in XML?**
 
-**A.        ** E�CFR files in XML have been converted and simplified from data in SGML that originates at the OFR.  The E�CFR is available in XML starting with the year 2015. Only the most recent e-CFR data will be available at endpoints on the FDsys Bulk Data Repository. Please see [www.ecfr.gov](http://www.ecfr.gov/) to search and browse a point in time edition of the e-CFR.
+**A.        ** E–CFR files in XML have been converted and simplified from data in SGML that originates at the OFR.  The E–CFR is available in XML starting with the year 2015. Only the most recent e-CFR data will be available at endpoints on the FDsys Bulk Data Repository. Please see [www.ecfr.gov](http://www.ecfr.gov/) to search and browse a point in time edition of the e-CFR.
```
Comment (LamLamLLC) [inline ECFR-XML-User-Guide.md:39]: [www.ecfr.gov](http://www.ecfr.gov/) to search and browse a point in time edition of the e-CFR.

File: ECFR-XML-User-Guide.md [hunk 2]
```diff
@@ -2277,8 +2277,8 @@ This is a Table Row element.  It is a sub-element of `<TABLE>`.
 This element is used to create cross references. It has both an identifier and a reference identifier.
 
 ```
-<DIV8 N="� 1.36" NODE="37:1.0.1.1.1.2.69.5" TYPE="SECTION">
-  <HEAD>� 1.36   Revocation of power of attorney; withdrawal of patent attorney or agent.</HEAD>
+<DIV8 N="§ 1.36" NODE="37:1.0.1.1.1.2.69.5" TYPE="SECTION">
+  <HEAD>§ 1.36   Revocation of power of attorney; withdrawal of patent attorney or agent.</HEAD>
```
Comment (LamLamLLC) [inline ECFR-XML-User-Guide.md:2281]: +  <HEAD>§ 1.36   Revocation of power of attorney; withdrawal of patent attorney or agent.</HEAD>

**Comment by jonquandt** [comment]
no further changes on this pull request.

**Pull request closed without merging**